### DISCUSSION TITLE
ALGÚN EQUIPO DISPONIBLE O CREAMOS UNO :)

### DISCUSSION THREAD

**Discussion by AnaMejiaR** [Launch-X-Latam/MisionFrontEnd#572] [Team Posts]
Hola a todos, alguna vacante para unirme a su equipo?.

O también podemos crear equipo si alguien está interesado, lx15797@innovaccion.mx

**Comment by Dataday28** [comment]
Yo estoy disponible para crear equipo,  lx03762@innovaccion.mx

**Comment by brayandoria98** [comment]
Yo estoy disponible. Mi correo es: lx10110@innovaccion.mx

**Comment by otrinidadv** [comment]
Hola si todavía hay lugar me uno, mi correo es lx02654@innovaccion.mx

**Comment by innovategm** [comment]
Hola, si hay algun lugar, mi correo es lx05800@innovaccion.mx

**Comment by BrandonAzure** [comment]
A mi me gustaría pertenecer a un equipo lx00326@innovaccion.mx

**Comment by JuanRodriguezDV** [comment]
lx08166@innovaccion.mx yo me quiero unir

**Comment by RogelioLR** [comment]
Hola!, por si queda un espacio, busco equipo, lx03656@innovacion.mx

**Comment by MatiasBazan** [comment]
Si aún tienen espacio en el equipo aquí dejo mí correo saludos !!! lx08634@innovaccion.mx

**Comment by isucresber** [comment]
Hola si aun tienen espacio en el equipo me gustaria unirme, dejo  mi correo lx05177@innovaccion.mx
